tests/exposure.py

```diff
@@ -56,20 +56,24 @@
 
 pexLog.Debug("lsst.afw.image", VERBOSITY)
 
-dataDir = os.path.join(lsst.utils.getPackageDir("afwdata"), "data")
-
-InputMaskedImageName = "871034p_1_MI.fits"
-InputMaskedImageNameSmall = "small_MI.fits"
-InputImageNameSmall = "small"
-OutputMaskedImageName = "871034p_1_MInew.fits"
-
-currDir = os.path.abspath(os.path.dirname(__file__))
-inFilePath = os.path.join(dataDir, InputMaskedImageName)
-inFilePathSmall = os.path.join(dataDir, InputMaskedImageNameSmall)
-inFilePathSmallImage = os.path.join(dataDir, InputImageNameSmall)
+try:
```

Comment:
I would prefer a smaller `try` block with an `else` clause. If you do that (and elsewhere) you could probably avoid use `pex_exceptions` as you'd only be wanting to catch any exception associated with trying to get the `dataDir` rather than the possibility of other parts of the `try` block throwing exceptions.